pkg/cosign/verify_test.go

```diff
@@ -1377,30 +1379,8 @@ func TestVerifyRFC3161Timestamp(t *testing.T) {
 		base64.StdEncoding.EncodeToString(signature),
 		static.WithCertChain(pemLeaf, appendSlices([][]byte{pemRoot})),
 		static.WithRFC3161Timestamp(&rfc3161TS))
-	err = VerifyRFC3161Timestamp(ociSig, pool)
+	_, err = VerifyRFC3161Timestamp(ociSig, pool)
 	if err == nil || !strings.Contains(err.Error(), "hashed messages don't match") {
 		t.Fatalf("expected error verifying mismatched signatures, got: %v", err)
 	}
-
-	// failure with old certificate
```

Comment:
FYI asked offline: this is removed because VerifyRFC3161Timestamp doesn't check cert expiration time. it only checks the TS